src/geovista/geodesic.py

```diff
@@ -576,20 +577,20 @@ def boundary(
 
         Examples
         --------
-        Add the boundary of the bounding-box to the plotter for the ``africa`` panel
-        of a cubed-sphere. The geodesic bounding-box is generated from the 4 corners of
-        the cubed-sphere panel located over Africa.
+        Add the boundary of a ``C32`` bounding-box to the plotter for a region
+        around the ``0`` meridian. The geodesic bounding-box is generated by
```

Comment:
@trexfeathers Being a stickler for consistency, could you align the description to be the same as lines 282-284, or vice versa i.e., use `around North Africa and Western Europe` or the above in both 👍

Reply:
It's not the same region

Reply:
I was actually wondering whether I should deliberately use different lats and lons for every corner just to demonstrate that it could be done

Reply:
Ahhh ...

I kinda wanted those examples to be themed, so they're all using the exact same region of extraction so that the user can easily see the difference in what's happening, hence why I used the `africa` panel.

So rather than using the `africa` panel, could we use the same `BBox` region though-out please.

Cheers

Reply:
Sounds good. I will find a single example that fits the bill (no pun intended) for all the cases. My smaller region that was good for demonstrating the 3D shape of `BBox` was also almost all masked data so that won't do!

Reply:
ac80b5665a4b743c119f4e658eeb43b479db0282